doc/.sphinx/.wordlist.txt

```diff
@@ -308,3 +308,5 @@ Zettabyte
 ZFS
 zpool
 zpools
+queueing
```

Comment:
```suggestion
```
Let's go for "queuing" instead - both are correct, but afaics aspell knows "queuing" but not "queueing".

Reply:
Fixed, thanks!